scripts/ios/build-sargon.sh

```diff
@@ -45,6 +45,18 @@ generate_ffi() {
     local TARGET_FOR_DYLIB_PATH="aarch64-apple-ios"
   fi 
   cargo run --features build-binary --bin sargon-bindgen generate --library target/$TARGET_FOR_DYLIB_PATH/release/lib$1.dylib --language swift --out-dir target/uniffi-xcframework-staging
+
+  if $release; then
+    echo "🚢 Ensuring Sargon build for release - that it will work for e.g. iOS wallet to archive."
```

Comment:
We must try the `swift build -c release` after FFI bindgen has run, and we try it before we create the XCFramework.